.github/ISSUE_TEMPLATE/onboarding_to_cluster.yaml

```diff
@@ -0,0 +1,137 @@
+name: Onboard to a cluster
+description: Get access to an Operate-First managed cluster.
+title: "NEW PROJECT: <name>"
+labels: [kind/onboarding, area/cluster]
+assignees: [first-operator]
+body:
+    - type: dropdown
+      id: cluster
+      attributes:
+          label: Target cluster
+          description: |
+              Please select a cluster from the list at
+              https://github.com/operate-first/apps/tree/master/cluster-scope/overlays/moc#available-clusters
+          options:
+              - Zero
+              - Infra
+              - Rick
+    - type: input
+      id: team-name
+      attributes:
+          label: Team name
+          description: |
+              Your team name should be a short identifier consisting of lower
+              case letters, numbers, and dashes (technically, anything thing
+              satisfies [RFC 1123](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names)). For example, `widget-research` is a
+              valid team name, but `Widget Research` is not, because it contains
+              whitespace.
+
+              We will use this name to create an OpenShift `group` for you and
+              your team members.
+
+              You need to pick a "team name" whether or not you are requesting
+              access for a group of people or an individual. We recommend that
+              you do not use your individual name as a team name.
+
+          placeholder: my-team
+      validations:
+          required: true
+    - type: input
+      id: project-name
+      attributes:
+          label: Desired project names
+          description: |
+              Resources in OpenShift are contained in "namespaces" (also called
+              "projects"). Unless you request otherwise, we will create a single
+              project for you with the same name as your selected team name.
+
+              If you need additional namespaces, or want a project name that
```

Comment:
for later parsing, can we update this to say something like "if you need more than one namespace, please list them here, using a comma delimiter